### PR TITLE
docs: Distributed DVT Cloud Run Jobs sample

### PR DESCRIPTION
**Pull request by nehanene15**
Sample code to run DVT in a distributed manner using Cloud Run and documentation clean up.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -469,19 +469,17 @@ View the complete YAML file for a Grouped Column validation on the
...
+When running DVT in a distributed fashion, both the `--kube-completions` and `--config-dir` flags are required. The `--kube-completions` flag specifies that the validation is being run in indexed completion mode in Kubernetes or as multiple independent tasks in Cloud Run. If the `-kc` option is used and you are not running in indexed mode, you will receive a warning and the container will process all the validations sequentially. If the `-kc` option is used and a config directory is not provided (a `--config-file` is provided instead), a warning is issued.
```
Comment (nj1973) [inline README.md]: Do we need en "i.e. " on the start of "a `--config-file` is provided instead"?

File: docs/internal/distributed_jobs.md [hunk 2]
```diff
@@ -1,22 +1,28 @@
...
-1. Build the container image with the database connection parameters. This is not recommended because the container image is specific to the databases being used OR
-2. Build a DVT container image as specified in the [Docker samples directory](../../samples/docker/README.md). When running the container, pass the `PSO_DV_CONN_HOME` environment variable to point to a GCS location containing the connection configuration. This is the suggested approach.
 
-## Future Work  
+1. Build the container image with the database connection parameters. This is not recommended because the container image would be limited to the databases being used.
+1. Build a DVT container image as specified in the [Cloud Run Jobs samples directory](../../samples/cloud_run_jobs/README.md). When creating a Job, specify the `PSO_DV_CONN_HOME` environment variable to point to a GCS location containing the connection configuration. This is the suggested approach.
```
Comment (nj1973) [inline docs/internal/distributed_jobs.md]: Is it right that "Job" in "When creating a Job" is capitalized?

File: docs/internal/distributed_jobs.md [hunk 3]
```diff
@@ -1,22 +1,28 @@
...
+
+## Future Work
+
 ### Use GCP secret manager to store database connection configuration
```
Comment (nj1973) [inline docs/internal/distributed_jobs.md]: I think "secret manager" in this case is the name of a service and therefore should be capitalized.

**Review by nj1973 — APPROVED** [review]
Just a few minor comments, LGTM otherwise.

**Review by Raniksingh — APPROVED** [review]

**Review by nj1973 — APPROVED** [review]
LGTM

**Pull request merged**